liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/InstallFeatureSupport.java

```diff
@@ -144,7 +144,7 @@ public File downloadArtifact(String groupId, String artifactId, String type, Str
                 String coordinate = String.format("%s:%s:%s",
                         dependencyArtifact.getGroupId(), FEATURES_JSON_ARTIFACT_ID, dependencyArtifact.getVersion());
                 result.add(coordinate);
-                log.info("Additional Json: " + coordinate);
+                log.info("Additional user feature json repsotiory: " + coordinate);
```

Comment:
repository is misspelled...but is this really a repository? Or just a coordinate for a json file?